nerdlandbot/commands/kerk.py

```diff
@@ -0,0 +1,30 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.parser import parse_channel
+
+from datetime import datetime
+
+class Kerk(commands.Cog, name="kerk"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="16ukerk", hidden=True)
+    async def cmd_kerk16(self, ctx: commands.Context, mention:typing.Optional[str] = None, *, message:typing.Optional[str] = None):
+        #check if message in lounge
+        if ctx.channel.name == ("lounge"):
+            #check if user mentioned  
+            if mention and mention[:2] == "<@":
+                #set a task to check if it's 16:00 and posts message
+                while not (datetime.now().hour == 16 and datetime.now().minute == 00):
+                    await asyncio.sleep(59)
+                msg = "⛪ Hey {0} het is 4u en <@{1}> wil u zien aan de kerk! 👊".format(mention,ctx.author.id)
+                if message:
+                    msg += "\n\t\"{0}\"".format(message)
+                return await ctx.send(msg)
```

Comment:
I think it would be better to send the message in botplayground, regardless of where the command was given. Since the message will not be an immediate reply to the command, I think it could annoy people if the message is in the lounge, especcially if there are multiple instances from that day.